library/std/src/sys_common/mod.rs

```diff
@@ -68,17 +68,17 @@ pub trait FromInner<Inner> {
     fn from_inner(inner: Inner) -> Self;
 }
 
-// Computes (value*numer)/denom without overflow, as long as both
```

Comment:
Ugh, another case of having to change a reasonable identifier.

Reply:
Okay.